### PR TITLE
Add gitignore icon

### PR DESCRIPTION
**Pull request by dirien**
Signed-off-by: Engin Diri <engin.diri@mail.schwarz>

**Issue:** #6834

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

see issue

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3628,6 +3628,11 @@
             "hex": "EA4AAA",
             "source": "https://github.com/sponsors"
         },
+        {
+            "title": "gitignore",
```
Comment (service-paradis) [inline _data/simple-icons.json:3632]: I think the name is `gitignore.io`. This is the name used everywhere on Github and in the doc.

**Review by service-paradis — CHANGES_REQUESTED** [review]
The icon is a perfect match 💯 Thank you @dirien for your contribution!
I'm not sure about the name as I mentioned in another comment. What do you think?

File: _data/simple-icons.json (hunk 1 above)
Comment (dirien) [reply _data/simple-icons.json]: Ok, i change it!

**Review by service-paradis — APPROVED** [review]
Great! 🎉

**Pull request merged**